src/vector/index.html

```diff
@@ -14,13 +14,13 @@
     <link rel="apple-touch-icon" sizes="180x180" href="vector-icons/apple-touch-icon-180x180.png">
     <link rel="manifest" href="manifest.json">
     <link rel="shortcut icon" href="vector-icons/favicon.ico">
-    <meta name="apple-mobile-web-app-title" content="Riot">
-    <meta name="application-name" content="Riot">
+    <meta name="apple-mobile-web-app-title" content="Tchap">
+    <meta name="application-name" content="Tchap">
     <meta name="msapplication-TileColor" content="#da532c">
     <meta name="msapplication-TileImage" content="vector-icons/mstile-144x144.png">
     <meta name="msapplication-config" content="vector-icons/browserconfig.xml">
     <meta name="theme-color" content="#ffffff">
-    <meta property="og:image" content="<%= htmlWebpackPlugin.options.vars.og_image_url %>" />
+    <meta property="og:image" content="https://chat.status.im/img/logos/riot-im-logo-1.png" />
```

Comment:
Is this meant to be a tchap logo?

Reply:
I dont know, the change comme from an old version of this file. 
Now, it should be `<%= htmlWebpackPlugin.options.vars.og_image_url %>`